onnx_xla/translations/translate_batch_normalization.cc

```diff
@@ -0,0 +1,25 @@
+#include "onnx_xla/operator_registry.h"
+
+namespace onnx_xla {
+
+// Tranlates batch normalization in test mode
+onnxStatus translateBatchNormalization(const Node& n,
+                                       XlaBuilder& builder,
+                                       ValueOpMap& valueToOp) {
+  if (n.outputs().size() > 1 || n.hasAttribute(kmomentum) ||
+      n.hasAttribute(Symbol("spatial"))) {  // TODO: ENFORCE
+    throw std::runtime_error("Only test mode of BatchNormalization supported");
+  }
+
+  float epsilon = 1e-5;
+  if (n.hasAttribute(kepsilon)) {
```

Comment:
where is `kepsilon` defined?

Reply:
kepsilon is one of the predefined ONNX symbols (found here https://github.com/onnx/onnx/blob/master/onnx/common/interned_strings.h). It's included from operator_registry.h through onnx/common/ir.h